static/elements/chromedash-feature-page.js

```diff
@@ -0,0 +1,264 @@
+        this.loading = false;
+        this.feature = feature;
+      }).catch(() => {
+      const toastEl = document.querySelector('chromedash-toast');
+      toastEl.showMessage('Some errors occurred. Please refresh the page or try again later.');
+    });
+  }
+
+  renderFeatureContent() {
+    return html`
+      ${this.feature.unlisted ? html`
+        <section id="access">
+        <p><b>This feature is only shown in the feature list to users with
+        edit access.</b></p>
+        </section>
+      `: nothing}
+   
+      ${this.feature.summary ? html`
+        <section id="summary">
+          <p class="preformatted">${autolink(this.feature.summary)}</p>
+        </section>
+      `: nothing}
+
+      ${this.feature.motivation ? html`
+        <section id="motivation">
+          <h3>Motivation</h3>
+          <p class="preformatted">${autolink(this.feature.motivation)}</p>
+        </section>
+      `: nothing}
+
+      ${this.feature.resources && this.feature.resources.samples ? html`
+        <section id="demo">
+          <h3>${this.feature.resources.length == 1 ? 'Demo' : 'Demos'}</h3>
+          <ul>
+            ${this.feature.resources.samples.map((sampleLink) => html`
+              <li><a href="${sampleLink}">${sampleLink}</a></li>
+            `)}
+          </ul>
+        </section>
+      `: nothing}
+
+      ${this.feature.resources && this.feature.resources.docs ? html`
+        <section id="documentation">
+          <h3>Documentation</h3>
+          <ul>
+            ${this.feature.resources.docs.map((docLink) => html`
+              <li><a href="${docLink}">${docLink}</a></li>
+            `)}
+          </ul>
+        </section>
+      `: nothing}
+
+      ${this.feature.standards.spec ? html`
+        <section id="specification">
+          <h3>Specification</h3>
+          <p><a href=${this.feature.standards.spec} target="_blank" rel="noopener">
+            Specification link
+          </a></p>
+          <br>
+          <p>
+            <label>Status:</label>
+            ${this.feature.standards.maturity.text}
+          </p>
+        </section>
+      `: nothing}
+    `;
+  }
+
+  renderFeatureStatus() {
+    return html`
+      <section id="status">
+        <h3>Status in Chromium</h3>
+        ${this.feature.browsers.chrome.blink_components ? html`
+          <p>
+            <label>Blink components:</label>
+            ${this.feature.browsers.chrome.blink_components.map((c) => html`
+              <a href="https://bugs.chromium.org/p/chromium/issues/list?q=component:${c}"
+               target="_blank" rel="noopener">${c}</a>
+            `)}
+          </p>
+        `: nothing}
+        <br>
+        <p>
+          <label>Implementation status:</label>
+          <b>${this.feature.browsers.chrome.status.text}</b>
+          ${this.feature.browsers.chrome.bug ? html`
+            <a href=${this.feature.browsers.chrome.bug} target="_blank" rel="noopener">
+              tracking bug
```

Comment:
The old version of the page used parenthesis: `(<a href=...>tracking bug</a>)`.  I don't think that it is a great visual treatment, but it does help prevent the user from incorrectly attempting to read the line of text like a sentence.

Reply:
Added!